apis/v1alpha1/instrumentation_types.go

```diff
@@ -145,6 +149,32 @@ type DotNet struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
 
+type ApacheHttpd struct {
+	// Image is a container image with Apache SDK and auto-instrumentation.
+	// +optional
+	Image string `json:"image,omitempty"`
+
+	// Env defines Apache HTTPD specific env vars. There are four layers for env vars' definitions and
+	// the precedence order is: `original container env vars` > `language specific env vars` > `common env vars` > `instrument spec configs' vars`.
+	// If the former var had been defined, then the other vars would be ignored.
+	// +optional
+	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Attrs defines Apache HTTPD agent specific attributes. The precedence is:
+	// `agent default attributes` > `instrument spec attributes`
+	// +optional
+	Attrs []corev1.EnvVar `json:"attrs,omitempty"`
+
+	// Apache HTTPD server version. One of 2.4 or 2.2. Default is 2.4
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
for my curiosity how is this version used?

Reply:
Apache HTTPD used these days is either version 2.2 or (most of the time) 2.4. The OTel agent is a set of binary loadable modules and those are compiled against specific Apache HTTPD version - different headers etc. Therefore you have to use the specific version of agent.

Reply:
Based on the version setting, auto-instrumentation configures Apache HTTPD with the proper set of modules to be loaded. The docker image with the agent contains modules for both supported versions.

Reply:
Pavol, 
could we move this forward, please?

Reply:
merged, thanks ❤️